connect/runtime/src/main/java/org/apache/kafka/connect/storage/ClusterConfigState.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.kafka.connect.runtime.distributed;
+package org.apache.kafka.connect.storage;
```

Comment:
Moved to a different package to leverage package-private fields and methods while testing other classes in this package (such as the `KafkaConfigBackingStore` class).

Reply:
`org.apache.kafka.connect.storage` is a public package, and AFAICS this change in public API wasn't mentioned in KIP-618.

Reply:
That package contains a mix of public and private API. The public parts are all documented in the [Javadocs](https://kafka.apache.org/32/javadoc/org/apache/kafka/connect/storage/package-summary.html) for the package and include the `Converter` interface and the `SimpleHeaderConverter` class; the private parts (which include [everything in the connect/runtime module](https://github.com/apache/kafka/tree/1d6e3d6cb3a798983cafb0367b90c3a51579c364/connect/runtime/src/main/java/org/apache/kafka/connect/storage)) contain the `ClusterConfigState` class.

Reply:
OK. I missed that this package move on its own won't make this class a public API.